website/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use stylist::{global_style, yew::styled_component};
+use yew::{html, Html};
+
+#[styled_component(Root)]
+fn root() -> Html {
+    global_style!(
+        r#"
+        html {
+            min-height: 100%;
+            position: relative;
+        }
+        
+        body {
+            height: 100%;
+            padding: 0;
+            margin: 0;
+        }
+    "#
+    )
+    .expect("Unable to mount global style");
+
+    let css = css!(
+        r#"
+        position: absolute;
+        overflow: hidden;
+        width: 100%;
+        height: 100%;
+        "#
+    );
+
+    html! {
+        <div class={ css }>
+            <canvas id="bevy_ggrs_example"></canvas>
+        </div>
+
+    }
+}
```

Comment:
What does yew give us, compared to just vanilla css in index.html?

Reply:
At the moment, nothing. I had been playing around with little landing page that allowed you to choose a "room" to pass to the app but stripped it back to be basics to try get it working. If we want to do something like that It'd be worth keeping, but if not then simple is preferable